cube/src/main/java/org/jboss/arquillian/ce/cube/CEEnvironmentProcessor.java

```diff
@@ -112,14 +150,15 @@ public void waitForDeployments(@Observes(precedence = -100) AfterStart event, Op
             log.warning(String.format("No environment for %s", testClass.getName()));
             return;
         }
-        log.info(String.format("Waiting for environment for %s", testClass.getName()));
```

Comment:
It's nice to have such a message somewhere, so that we know what's happening when waiting for stuff or debugging something.